public/js/get_events.js

```diff
@@ -0,0 +1,22 @@
+/* global XMLHttpRequest */
+var module = (function () {
+  console.log(module);
+  return {
+    getEvents: function (path, callback) {
```

Comment:
I would say this function should take `(queryParams, callback)` as arguments.

So e.g. I could write
```js
getEvents({date_from: '2017-09-01'}, someCallback)
```

Reply:
alright so send an object to the getEvents :+1:

Reply:
:+1: